pkg/lwip/include/lwip/sock_internal.h

```diff
@@ -69,7 +69,7 @@ static inline ssize_t lwip_sock_send(struct netconn *conn,
 
     return lwip_sock_sendv(conn, &snip, proto, remote, type);
 }
-/**
+/** @internal
```

Comment:
Is this needed?

Reply:
sadly yes (and i dont know weather doxgen considers that bug or feature) 
`@internal` removes the rest of that comment (i think this is also for the "opening and closing of groups"-check` the result is that a internal open brace will not be counted and the closing one would be.